examples/flask/test_rock/app.py

```diff
@@ -73,6 +74,35 @@ def get_redis_database() -> redis.Redis | None:
     return g.redis_db
 
 
+def get_rabbitmq_connection():
+    """Get rabbitmq connection."""
+    if "rabbitmq" not in g:
+        if "RABBITMQ_HOSTNAME" in os.environ:
+            username = os.environ["RABBITMQ_USERNAME"]
+            password = os.environ["RABBITMQ_PASSWORD"]
+            hostname = os.environ["RABBITMQ_HOSTNAME"]
+            vhost = os.environ["RABBITMQ_VHOST"]
+            port = os.environ["RABBITMQ_PORT"]
+            credentials = pika.PlainCredentials(username, password)
+            parameters = pika.ConnectionParameters(hostname, port, vhost, credentials)
+            g.rabbitmq = pika.BlockingConnection(parameters)
+        else:
+            return None
+    return g.rabbitmq
+
+
+def get_rabbitmq_connection_from_uri():
+    """Get rabbitmq connection from uri."""
+    if "rabbitmq_from_uri" not in g:
+        if "RABBITMQ_CONNECT_STRING" in os.environ:
+            uri = os.environ["RABBITMQ_CONNECT_STRING"]
+            parameters = pika.URLParameters(uri)
+            g.rabbitmq_from_uri = pika.BlockingConnection(parameters)
```

Comment:
Same here

Reply:
The connection is stored in the flask application context [g](https://flask.palletsprojects.com/en/2.3.x/api/#flask.g). I did that mainly for consistency with the previous functions.